nrf_modem/include/nrf_modem_dect_mac.h

```diff
@@ -0,0 +1,2891 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA. All Rights Reserved.
```

Comment:
Not related to this file, but in overall, not just for dect, I'd like to have nrf_modem_version.h exported in this folder as well. It is not straightforward to see the used libmodem afterwards from github.